src/native/third_party/isa-l.gyp

```diff
@@ -150,7 +150,18 @@
                 'isa-l_crypto/md5_mb/md5_mb_mgr_flush_avx512.asm',
                 'isa-l_crypto/md5_mb/md5_mb_x16x2_avx512.asm',
                 'isa-l_crypto/md5_mb/md5_ctx_avx512.c',
-            ]}]],
+            ]}, 'node_arch=="arm64"', {'sources': [
+                'isa-l_crypto/md5_mb/aarch64/md5_ctx_aarch64_asimd.c',
+                'isa-l_crypto/md5_mb/aarch64/md5_mb_aarch64_dispatcher.c',
+                'isa-l_crypto/md5_mb/aarch64/md5_mb_mgr_aarch64_asimd.c',
+                'isa-l_crypto/md5_mb/aarch64/md5_mb_asimd_x4.S',
```

Comment:
I don't think if gyp has a builtin rule for .S or we need a custom rule for it like we do for .asm in:
https://github.com/noobaa/noobaa-core/blob/master/src/native/asm.gypi

See https://stackoverflow.com/a/51110745/2702164
I would look in the Makefile.am/make.inc if gcc handles it or yasm/nasm.

Reply:
$(O)/%.o: $(host_cpu)/%.S
	@echo "  ---> Building $< $(msg)"
	@$(AS) $(ASFLAGS) -o $@ $<

indeed yasm/nasm

Reply:
But in this case AS is gcc and gcc passes to gas.

    ifeq ($(arch),aarch64)
        AS=$(CC) -D__ASSEMBLY__

Reply:
@gbtucker does ISA-L support both nasm/yasm/gas all the same?
Can we use gas to build on all archs?
Is there any assembler we should prefer to use for ISA-L?

Reply:
For x86, nasm is the most up-to-date.  Nasm can build the x86 code for unix or windows.  The arm optimizations were written for gas so I think you must use gcc when building for arm unix.  You can mix other compilers for x86; gcc, icc, vc, etc.

Reply:
thank you very much @gbtucker!